be/src/Project/Services/Analytics/Analytics.Api/Services/SalesAnalytics/SalesService.cs

```diff
@@ -219,9 +219,14 @@ select new
 
         public IEnumerable<CountrySalesServiceModel> GetCountrySales(GetCountriesSalesServiceModel model)
         {
+            var endDate = DateTime.UtcNow;
+            var startDate = endDate.AddMonths(-2);
```

Comment:
**suggestion:** exactly this we need to avoid. We should inject it dynamically via parameters

Reply:
I know and remember.  This will be in the charts-data-range branch